twisted/words/protocols/jabber/ijabber.py

```diff
@@ -41,9 +41,10 @@ class IIQResponseTracker(Interface):
     naturally with deferreds. You send out a request and when the response
     comes back a deferred is fired.
 
-    The L{IQ} class implements a C{send} method that returns a deferred. This
-    deferred is put in a dictionary that is kept in an L{XmlStream} object,
-    keyed by the request stanzas C{id} attribute.
+    The L{<twisted.words.protocols.jabber.client.IQ>} class implements a C{send}
```

Comment:
this should probably be changed as no text is written for the link. http://epydoc.sourceforge.net/epytext.html see section 3.3